marlowe-runtime/web/Language/Marlowe/Runtime/Web/Orphans.hs

```diff
@@ -0,0 +1,415 @@
+{-# LANGUAGE OverloadedLists #-}
+
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Language.Marlowe.Runtime.Web.Orphans
```

Comment:
Is it possible to unit test this module such that a programmer who isn't familiar with this code (eg us in three months 😉) won't leave the schema out of date?

Reply:
Yes - all the schemas are tested in `web-server-test`